elk/training/train.py

```diff
@@ -83,11 +83,12 @@ def apply_to_layer(
             reporter = CcsReporter(self.net, d, device=device, num_variants=v)
             train_loss = reporter.fit(first_train_h)
 
-            (_, v, k, _) = first_train_h.shape
-            reporter.platt_scale(
-                to_one_hot(repeat(train_gt, "n -> (n v)", v=v), k).flatten(),
-                rearrange(first_train_h, "n v k d -> (n v k) d"),
-            )
+            if self.net.norm == "leace":
```

Comment:
I would slightly prefer `if self.net.norm != "burns"` here for the same reasons of extensibility.